tests/trig.rs

```diff
@@ -426,15 +636,86 @@ mod test {
         ]);
         assert_eq!(terms.force().len(), 7);
 
-        let rules = Trig::run_workload(
-            terms,
-            complex,
-            Limits {
-                iter: 3,
-                node: 2000000,
-            },
-        );
+        let mut all = complex;
+        all.extend(prior_forbidden_rules());
+
+        let rules = Trig::run_workload(terms, all, limits);
+        assert_eq!(rules.len(), 6);
+    }
+
+    #[test]
+    fn sandbox() {
```

Comment:
wait what is this ruleset from?

Reply:
Oh sorry this was for building the recipe quickly